docs/file-reference.md

```diff
@@ -141,6 +143,7 @@ Each "app" (Kedge file) is a Kubernetes <a target="_blank" href="https://kuberne
 | ingresses | array of [ingress object](#ingressobject) | no           | [ingress object](#ingressobject) object |
 | routes | array of [route object](#routeobject) | no           | [route object](#routeobject)  object |
 | secrets | array of [secret](#secret) | no           | [secret](#secret) object |
+| imageStreams | array of [imageStream object](#imageStreamObject) | no           | [imageStream object](#imageStreamObject)  object |
```

Comment:
link reference has to be lowercase `[imageStream object](#imagestreamobject) `

Reply:
fixed